client/models/service-instance.js

```diff
@@ -2,15 +2,6 @@ var request = require('request');
 var util = require('util');
 
 module.exports = function(ServiceInstance) {
-  function applyPatch(target, fileData, callback) {
-    return this._appCommand({
-      cmd: 'patch',
-      target: target,
-      patch: fileData
-    }, callback);
-  }
-  ServiceInstance.prototype.applyPatch = applyPatch;
```

Comment:
Im changing a lot of the REST API in this PR. A major version bump is probably required.